overlord/devicestate/systems.go

```diff
@@ -96,11 +96,12 @@ func loadSeedAndSystem(label string, current *currentSystem) (seed.Seed, *System
 		return nil, nil, fmt.Errorf("cannot obtain brand: %v", err)
 	}
 	system := &System{
-		Current: false,
-		Label:   label,
-		Model:   model,
-		Brand:   brand,
-		Actions: defaultSystemActions,
+		Current:               false,
+		Label:                 label,
+		Model:                 model,
+		Brand:                 brand,
+		Actions:               defaultSystemActions,
+		DefaultRecoverySystem: defaultRecoverySystem != "" && defaultRecoverySystem == label,
```

Comment:
this vs the code below makes me wonder if we need to capture more than just a label about the default system?

Reply:
When we do something like rebooting into a new system, we only pick a system by a single label. What else would we need to determine which system is the default system?

Reply:
I know and in practice if the system is tampered with unsealing won't work if FDE is setup, I still think that it would be more consistent to capture as much information as we do for the entries in seeded-systems for the default one and do a similar kind of comparison if that makes sense?